web/src/cypress/integration/admin.ts

```diff
@@ -30,29 +30,49 @@ function testAdmin(screen: ScreenFormat) {
 
     it('should update a config value', () => {
       const newVal = 'http://' + c.domain()
+
       cy.get('input[name="General.PublicURL"]')
         .clear()
+        .should('be.empty')
         .type(newVal)
+        .should('have.value', newVal)
       cy.get('button[data-cy="save"]').click()
-      cy.get('p[data-cy="old"]').should('contain', cfg.General.PublicURL)
-      cy.get('p[data-cy="new"]').should('contain', newVal)
+
+      // save dialog
+      cy.get(
+        'ul[data-cy="confirmation-diff"] li[data-cy="General.PublicURL"] p[data-cy="old"]',
```

Comment:
nit: should we drop the double quotes?

Reply:
It doesn't work with `General.PublicURL` because of the period. I've found not needing double quotes to be pretty random on certain characters, so I always use quotes so I don't have to think/test what will and won't work